retail/solutions/README.md

```diff
@@ -1,14 +1,16 @@
 # Retail Solutions
```

Comment:
@rocavalc , as discussed during our chat last night, let's change this to Unified Customer Profile (UCP).
UCP will be the parent topic and within this, we can have a separate section to cover "Retail channel churn prediction". This structure will be in line with the actual composition of UCP. Thanks.